README.md

```diff
@@ -137,12 +143,14 @@ If you like this project send thanks to Microverse.
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-I would like to thank my code reviewer for showing me this README template.
+Special thanks to https://unsplash.com/ and https://www.pexels.com/ (along with their photographers).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Please make sure your MIT license link work. You need to create a file MIT.md and add your license text for to make it work. Your mit link currently doesn't work:
![image](https://user-images.githubusercontent.com/74411135/235274230-0120ea38-f64f-406c-8158-30768682612d.png)

This will help you get full ownership on your project 😃

Reply:
Hi @Trast00 
Thank you so much for your review. GitHub was crashing earlier today. Would you mind double-checking this? MIT.md is working on my GitHub.